src/Microsoft.IdentityModel.Clients.ActiveDirectory/TokenCache.cs

```diff
@@ -628,9 +631,9 @@ internal AdalResultWrapper LoadFromCacheCommon(CacheQueryData cacheQueryData, Re
             }
         }
 
-        private static bool IsSameCloud(string authority, string authority1)
+        private static bool IsSameCloud(string cachedAuthority, string cacheQueryDataAuthority)
```

Comment:
>cacheQueryDataAuthority [](start = 71, length = 23)

it's not really obvious (to me) what this is. I'm assuming this is the value which was provided by the user/developer as a parameter?